app/dashboard/notifications.py

```diff
@@ -42,6 +43,31 @@
 logger = logging.getLogger(__name__)
 
 
+def notify_of_lowball_bounty(bounty):
+    """Send an email to founders@gitcoin.co with the lowball bounty info
+
+    Args:
+        bounty (dashboard.models.Bounty): The lowball bounty object
+
+    """
+    to_email = 'founders@gitcoin.co'
+    from_email = settings.CONTACT_EMAIL
+    cur_language = translation.get_language()
+    try:
+        setup_lang(to_email)
+        subject = _("Low Bounty Notification")
+        body = \
+            f"Bounty: {bounty}\n\n" \
+            f"url: {bounty.url}\n\n" \
+            f"Owner Name: {bounty.bounty_owner_name}\n\n" \
```

Comment:
having the amount in here would sure be nice :)
also it should probably only happen if bounty is on mainnet